docs/rawdata/raw_data_download.md

```diff
@@ -0,0 +1,178 @@
+Input "terminal" into the text field and select the terminal application from the suggestions. 
+
+![data_download_mac_command_line.png](images/data_download_mac_command_line.png)
+
+#### Open command line on Windows
+
+Click on the magnifying glass in the toolbar at the bottom of the screen. 
+Input "terminal" into the search field and select the terminal application from the suggestions.
+
+![data_download_windows_command_line.png](images/data_download_windows_command_line.png)
+
+#### Download data via command line
+
+We show how to run the download via the two popular command line clients [cURL](https://curl.se/docs/manpage.html) and [GNU Wget](https://www.gnu.org/software/wget/).
+Of course, you can use any other software that supports HTTP.
+
+!!! warning
+    The data will be downloaded into the directory in which the command is run.
+    Please ensure that it's the correct directory and there is enough space before running the download
+
+The commands contain two placeholders, where you have to provide your custom input:
+
+- *ACCESS_TOKEN*: your generated [personal access token](#personal-access-token)
+- *MEASUREMENT_URL*: the [URL](#raw-data-url-generation) associated with the data of the measurement
+
+=== "curl"
+
+    ``` bash
+    curl -OJ -H "Authorization: Bearer <ACCESS_TOKEN>" <MEASUREMENT_URL>
+    ```
+
+=== "wget"
+
+    ``` bash
+    wget --content-disposition --trust-server-names --header "Authorization: Bearer <ACCESS_TOKEN>" <MEASUREMENT_URL>
+    ```
+For example to download the raw data associated with measurement **MSQ7645002AL-182987406699583** the commands would look as follows:
+
+=== "curl"
+
+    ``` bash
+    curl --parallel --fail -OJ -H "Authorization: Bearer v71E00f750Z78oBW4SKs90Vrd39h98eG" https://download.qbic.uni-tuebingen.de/measurements/MSQ7645002AL-182987406699583
+    ```
+
+=== "wget"
+
+    ``` bash
+    wget --content-disposition --trust-server-names --header "Authorization: Bearer v71E00f750Z78oBW4SKs90Vrd39h98eG"  https://download.qbic.uni-tuebingen.de/measurements/MSQ7645002AL-182987406699583
+    ```
+
+To download multiple measurements using one command, you can provide a text file containing one measurement URL per line (as in the one you can download in the raw data summary!).
+
+```txt
+<MEASUREMENT_URL_1>
+<MEASUREMENT_URL_2>
+```
+
+The corresponding commands need to be adapted accordingly.
+
+=== "curl"
+
+    ``` bash
+    curl --remote-name-all -OJ -H "Authorization: Bearer <ACCESS_TOKEN>" $(cat <file-with-urls>)
+    ```
+
+=== "wget"
+
+    ``` bash
+    wget --content-disposition --trust-server-names --header "Authorization: Bearer <ACCESS_TOKEN>" -i <file-with-urls>
+    ```
+
+!!! warning
+    To ensure character validity in the text file, please format it in the *UTF-8* format
+
+The easiest way to generate such a text file is to navigate into the raw data view and select the measurements for which you want to download the raw data for and click the "download url list" button.
```

Comment:
NIT: this is a repetition from the explanation above

Reply:
you may want to shorten it

Reply:
Good point i linked it to the section instead of repeating it